src/main/java/wooteco/subway/line/LineService.java

```diff
@@ -0,0 +1,48 @@
+package wooteco.subway.line;
+
+import org.springframework.stereotype.Service;
+import wooteco.subway.exception.LineNameDuplicatedException;
+import wooteco.subway.exception.LineNotFoundException;
+import wooteco.subway.line.dao.LineDao;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class LineService {
+
+    private final LineDao lineDao;
+
+    public LineService(LineDao lineDao) {
+        this.lineDao = lineDao;
+    }
+
+    public Line createLine(String name, String color) {
+        if (lineDao.findLineByName(name).isPresent()) {
```

Comment:
```suggestion
        if (existsByName(name)) {
```
find메서드 이후 isPresent 메서드를 호출하고 있는 부분을 existsByName 메서드로 추출해보면 어떨까요
추출한 메서드가 좀더 읽기 좋지 않을까요?

Reply:
추출해보았습니다~ㅎㅎㅎ